Source/WebKit/GPUProcess/media/RemoteMediaSourceProxy.cpp

```diff
@@ -83,12 +83,14 @@ const PlatformTimeRanges& RemoteMediaSourceProxy::buffered() const
     return m_buffered;
 }
 
-void RemoteMediaSourceProxy::seekToTime(const MediaTime& time)
+void RemoteMediaSourceProxy::seekToTarget(const WebCore::SeekTarget& target, CompletionHandler<void(const MediaTime&)>&& completionHandler)
 {
-    if (!m_connectionToWebProcess)
+    if (!m_connectionToWebProcess) {
+        completionHandler(MediaTime::invalidTime());
         return;
+    }
 
-    m_connectionToWebProcess->connection().send(Messages::MediaSourcePrivateRemote::SeekToTime(time), m_identifier);
+    m_connectionToWebProcess->connection().sendWithAsyncReply(Messages::MediaSourcePrivateRemote::SeekToTarget(target), WTFMove(completionHandler), m_identifier);
```

Comment:
Is MediaTime:: invalidTime() used if GPUProcess crashes in the middle of this async IPC?

Reply:
I don't know how IPC deals with process crashes with completion handler. This seems to be a more overall problem.

Reply:
Created a `AsyncReplyError<WTF::MediaTime>` template to define the default value when a connection failure occurs